data-gen/snoopAuditlog.py

```diff
@@ -80,6 +80,53 @@ def find_openapi_entry(openapi_spec, event):
   hit_cache[url.path] = None
   return None
 
+discovery_ops = [
+    'get/getCoreAPIVersions',
+    'get/getAPIVersions',
+    'get/getAppsV1beta2APIResources',
+    'get/getAdmissionregistrationV1beta1APIResources',
+    'get/getAuthenticationV1APIResources',
+    'get/getAuthenticationV1beta1APIResources',
+    'get/getAuthorizationV1APIResources',
+    'get/getAuthorizationV1beta1APIResources',
+    'get/getAutoscalingV1APIResources',
+    'get/getAutoscalingV2beta1APIResources',
+    'get/getAutoscalingV2beta2APIResources',
+    'get/getApiextensionsV1beta1APIResources',
+    'get/getApiregistrationV1APIResources',
+    'get/getApiregistrationV1beta1APIResources',
+    'get/getAppsV1APIResources',
+    'get/getAppsV1beta1APIResources',
+    'get/getAppsV1beta2APIResources',
+    'get/getBatchV1APIResources',
+    'get/getBatchV1beta1APIResources',
+    'get/getBatchV2alpha1APIResources',
+    'get/getCertificatesV1beta1APIResources',
+    'get/getCoordinationV1APIResources',
+    'get/getCoordinationV1beta1APIResources',
+    'get/getEventsV1beta1APIResources',
+    'get/getExtensionsV1beta1APIResources',
+    'get/getNodeV1beta1APIResources',
+    'get/getPolicyV1beta1APIResources',
+    'get/getSettingsV1alpha1APIResources',
+    'get/getSchedulingV1APIResources',
+    'get/getSchedulingV1beta1APIResources',
+    'get/getSchedulingV1alpha1APIResources',
+    'get/getStorageV1APIResources',
+    'get/getRbacAuthorizationV1APIResources',
+    'get/getRbacAuthorizationV1beta1APIResources',
+    'get/getStorageV1beta1APIResources',
+    'get/getNetworkingV1beta1APIResources',
+    'get/getNetworkingV1APIResources',
+    'post/createCoreV1Namespace',
+    'get/readCoreV1Namespace',
+    'post/createRbacAuthorizationV1beta1NamespacedRoleBinding',
+    'post/createAuthorizationV1beta1SubjectAccessReview',
+    'watch/listCoreV1NamespacedServiceAccount',
+    'get/readCoreV1NamespacedPod',
+    # 'post/createCoreV1NamespacedPod',
+]
+
```

Comment:
This list was generated by exploring common patterns within test_sequences.json.
They are primarily **get/get*APIResources** and the namespace / RBAC creation/watch surrounding each test.